ray-operator/controllers/ray/common/pod.go

```diff
@@ -119,6 +148,76 @@ func DefaultWorkerPodTemplate(instance rayiov1alpha1.RayCluster, workerSpec rayi
 	return podTemplate
 }
 
+func setLivenessProbe(probe *v1.Probe, rayNodeType rayiov1alpha1.RayNodeType) {
+	if rayNodeType == rayiov1alpha1.HeadNode {
+		// head node liveness probe
+		cmd := []string{
+			"bash", "-c", fmt.Sprintf("wget -q -O- http://localhost:%d/%s | grep success",
+				DefaultRayAgentPort, RayAgentRayletHealthPath),
+			"&&", "bash", "-c", fmt.Sprintf("wget -q -O- http://localhost:%d/%s | grep success",
+				DefaultRayDashboardPort, RayDashboardGCSHealthPath),
+		}
+		probe.Exec = &v1.ExecAction{Command: cmd}
+	} else {
+		// worker node liveness probe
+		cmd := []string{
+			"bash", "-c", fmt.Sprintf("wget -q -O- http://localhost:%d/%s | grep success",
+				DefaultRayAgentPort, RayAgentRayletHealthPath),
+		}
+		probe.Exec = &v1.ExecAction{Command: cmd}
+	}
+	if probe.InitialDelaySeconds != 0 {
```

Comment:
should we assign default value when the value == 0?

Reply:
if it is 0, kubernetes will use the default value.

Reply:
Should it be `if probe.InitialDelaySeconds == 0 {`?